File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/get-vetting-batches-by-country-id_param.ts


import { IGetVettingBatchesByCountryID_Param } from './iget-vetting-batches-by-country-id_param';

export class GetVettingBatchesByCountryID_Param implements IGetVettingBatchesByCountryID_Param {
  
	public CountryID: number = 0;
	public VettingBatchStatus: string = "";
	public IsCorrectionRequired?: boolean;
	public HasHits?: boolean;
	public CourtesyType: string = "";
	public AllHits?: boolean;
	public CourtesyStatus: string = "";
  
}

File: GTTS/INL.VettingService.Models/GetVettingBatchesByCountryID_Param.cs
namespace INL.VettingService.Models
{
    public class GetVettingBatchesByCountryID_Param : IGetVettingBatchesByCountryID_Param
    {
        public int CountryID { get; set; }
        public string VettingBatchStatus { get; set; }
        public bool? IsCorrectionRequired { get; set; }
        public bool? HasHits { get; set; }
        public string CourtesyType { get; set; }
        public bool? AllHits { get; set; }
        public string CourtesyStatus { get; set; }
    }
}